webpack/index.d.ts

```diff
@@ -689,6 +689,84 @@ declare namespace webpack {
         constructor(definitions: {[key: string]: any});
     }
 
+    class DllPlugin extends Plugin {
+        constructor(options: DllPlugin.Options | DllPlugin.Options[]);
+    }
+
+    namespace DllPlugin {
+        interface Options {
+            /**
+             * The absolute path to the manifest json file (output).
+             */
+            path: string,
```

Comment:
@hill0826 could you
1. change the type member delimiters to `;` to match the rest of the file
2. add tests [here](https://github.com/hill0826/DefinitelyTyped/blob/e4566cbfc15637e31930a90a9bcaa75b8d25421e/webpack/webpack-tests.ts#L21-L29)

i'd also order these keys alphabetically.